make.js

```diff
@@ -72,9 +72,7 @@ var MOCHA_OPTS = ' --timeout 15000 --slow 500';
     };
 
     target.tryStop = function() {
-        try {
-            exec(FOREVER + ' stop app.js');
-        } catch (e) { }
+        exec(FOREVER + ' stop app.js || true');
```

Comment:
@XhmikosR I know this is super annoying, but I'd probably do something similar in bash.